pages/gallery/[galleryId]/collection/create.tsx

```diff
@@ -0,0 +1,119 @@
+import { useCallback, useState } from 'react';
+import CollectionEditorProvider, {
+  useCollectionMetadataState,
+  useStagedCollectionState,
+} from 'contexts/collectionEditor/CollectionEditorContext';
+import { useModalActions } from 'contexts/modal/ModalContext';
+import CollectionWizardContext from 'contexts/wizard/CollectionWizardContext';
+import { useTrack } from 'contexts/analytics/AnalyticsContext';
+import { graphql, useLazyLoadQuery } from 'react-relay';
+import CollectionCreateOrEditForm from 'flows/../../src/components/ManageGallery/OrganizeCollection/CollectionCreateOrEditForm';
+import CollectionEditor from 'flows/../../src/components/ManageGallery/OrganizeCollection/Editor/CollectionEditor';
+import FullPageCenteredStep from 'flows/../../src/components/Onboarding/FullPageCenteredStep/FullPageCenteredStep';
```

Comment:
these filepaths lol 🤯

Reply:
hmmm, is there some eslint plugin to make this shit nicer? this is so ugly but just what intellij did lol

Reply:
actually those paths were fucked and incorrect. intellij goofed during a file move refactor.